PaulaPerazzo/dalle-react
# feature(develop): nothing

## What I did

- x coisa

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout main`
   1. `git pull`
   1. `git checkout develop`
   1. `yarn install` ou npm install
   1. `yarn start`

1. In your browser go to <http://localhost>:port

1. Check if the component looks just like the mockup in different screen sizes

1. Read the changed files section on GitHub

1. Test the things that you think are worth testing, even the ones that are not in this description
